main_refactor.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+import argparse
+
+import header_mapping as hm
+from operators import process_csv
+from agol_connection import AGOLConnection
+from ingester import Ingester
+
+
+def process_instantaneous(dry_run=False, datadir=None):
```

Comment:
This is much cleaner :)

Reply:
Thanks, and I think sets things in a good trajectory. I had originally just put a new function in `main.py` called `process_instantenous_refactor()`, but had to abandon that due to the circular imports it required.